django/db/backends/postgresql/base.py

```diff
@@ -144,6 +145,36 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._named_cursor_idx = 0
 
+    @classmethod
+    def config_from_url(cls, engine, scheme, url):
+        parsed = parse.urlparse(url)
+        host, _ = parsed._hostinfo
```

Comment:
_ is used elsewhere for gettext. Maybe better to just take the [0] slice instead?

Reply:
👍